src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx

```diff
@@ -51,7 +50,7 @@ export const MarketingLandingApp = () => {
         <MeetTheSpecialists />
         <CollectorsOverview />
         <PreviouslySoldOnArtsyRailQueryRenderer />
-        <Reviews />
+        {/* <Reviews /> */}
```

Comment:
commenting them here instead of removing this line to make sure no one rebuilds the component by mistake in the future